tests/plugins/search_relevance/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM opensearchstaging/opensearch:3.1.0
 
+RUN /usr/share/opensearch/bin/opensearch-plugin remove opensearch-search-relevance
```

Comment:
This one is odd. The plugin is included in the distribution? Why does it need to be uninstalled?